IFC4x3/Properties/p/PortCapacity_2wGIZukqb5y8_rmxPBWB8h/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PortCapacity_2wGIZukqb5y8_rmxPBWB8h" Name="PortCapacity" UniqueId="ba4128f8-bb49-45f0-8fb5-c3b64b80b22b" PrimaryDataType="IfcInteger" />
```

Comment:
Is this not expected to be modeled?

Reply:
This is not expected to be modeled by IfcDistributionPort. This only states port connections that can be supported by the distribution frame. It is fixed information for the product.